packages/teams-js/src/public/meeting.ts

```diff
@@ -396,7 +396,7 @@ export namespace meeting {
   /**
    * @hidden
    * Allows an app to get the authentication token for the anonymous or guest user in the meeting
-   *
+   * Is valid for side panel, meeting stage and task module
```

Comment:
We're referring to "task modules" as "dialogs" now (as far as external developers are concerned). Please change task module to dialog